### ISSUE TITLE
Make codeblocks non-executable

### ISSUE THREAD

**Issue by jlin27** [pytorch/tutorials#792]
Certain codeblocks in the [Training a Classifier colab](https://colab.research.google.com/github/pytorch/tutorials/blob/gh-pages/_downloads/cifar10_tutorial.ipynb#scrollTo=N1BibYLW8C4N) should be non-executable; otherwise, users will run and expect output and be confused when they get an error. Specifically the code under the "2. Define a Convolution Neural Network" and the  "3. Define a Loss function and optimizer" sections. 

They should match the html version: https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py
